### PR TITLE
Update Cross-VM bridge docs

### PR DESCRIPTION
**Pull request by sisyphusSmiling**
Closes: #807 #878

### CODE REVIEW

File: docs/evm/vm-bridge.md [hunk 1]
```diff
@@ -6,8 +6,18 @@ sidebar_position: 7
 
 # Cross-VM Bridge
 
-Flow provides the cross-VM bridge which enables Cadence to interact with EVM on Flow in order to bridge tokens, fungible and non-fungible, from one VM environment to the other. The cross-VM bridge is a contract-based, trustless protocol enabling the automated bridging of tokens from Cadence into EVM with  the corresponding ERC-20 and ERC-721 token types. In the opposite direction, it supports bridging of arbitrary EVM ERC-20 and ERC-721 tokens into the corresponding Cadence FT or NFT token types. The cross-VM bridge internalizes the capabilities to deploy new token contracts in either VM state as needed, resolving access to, and maintaining links between , contracts. It additionally automates account and contract calls to enforce source VM asset burn or lock, and target VM token mint or unlock.
+Flow provides the cross-VM bridge which enables the movement of fungible and non-fungible tokens between Cadence & EVM.
```
Comment (gregsantos) [inline docs/evm/vm-bridge.md:9]: Should Cross-VM Bridge be proper (capitalized)? Especially if it's "The" crossVM bridge?
Or perhaps The Flow Cross-VM Bridge?

File: docs/evm/vm-bridge.md [hunk 2]
```diff
@@ -6,8 +6,18 @@ sidebar_position: 7
...
+The cross-VM bridge is a contract-based, trustless protocol enabling the automated and atomic bridging of tokens from
+Cadence into EVM with their corresponding ERC-20 and ERC-721 token types. In the opposite direction, it supports
+bridging of arbitrary ERC-20 and ERC-721 tokens from EVM to Cadence as their corresponding FT or NFT token types.
 
-Developers wishing to use the cross-VM bridge will be required to use a Cadence transaction as cross-VM bridging functionality is currently not available natively in EVM.
+The cross-VM bridge internalizes the capabilities to deploy new token contracts in either VM state as needed, resolving
+access to, and maintaining links between associated contracts. It additionally automates account and contract calls to
+enforce source VM asset burn or lock, and target VM token mint or unlock.
+
+Developers wishing to use the cross-VM bridge will be required to use a Cadence transaction as cross-VM bridging
```
Comment (gregsantos) [inline docs/evm/vm-bridge.md:18]:
```suggestion
Developers wishing to use the cross-VM bridge will be required to use a Cadence transaction for cross-VM bridging
```

**Comment by sisyphusSmiling** [comment]
Side note - it'd be really nice to sync snippet content with source code so that updates to the code are reflected in docs immediately. Something like [this docusaurus plugin](https://github.com/christian-bromann/docusaurus-theme-github-codeblock) might fill the gap. Even better if we could extend it serve as a transaction/script a runner in page.

File: docs/evm/cadence/vm-bridge.md [hunk 3]
```diff
@@ -0,0 +1,911 @@
...
+Works](../how-it-works.md).
+
+:::
+
+### Overview
+
+The Flow EVM bridge allows both fungible and non-fungible tokens to move atomically between Cadence and EVM. In the
+context of EVM, fungible tokens are defined as ERC20 tokens, and non-fungible tokens as ERC721 tokens. In Cadence,
+fungible tokens are defined by contracts implementing FungibleToken and non-fungible tokens the NonFungibleToken
+standard contract interfaces.
```
Comment (joshuahannan) [inline docs/evm/cadence/vm-bridge.md:64]:
```suggestion
fungible tokens are defined by contracts implementing 
[the `FungibleToken` interface](https://github.com/onflow/flow-ft/blob/master/contracts/FungibleToken.cdc)
and non-fungible tokens implement
[the `NonFungibleToken` interface](https://github.com/onflow/flow-nft/blob/master/contracts/NonFungibleToken.cdc).
```

**Review by joshuahannan — APPROVED** [review]

**Pull request merged**